### PR TITLE
Change the compression to lz4 on main

### PR DESCRIPTION
**Pull request by larry-aptos**
## Type of Change
- [ ] New feature
- [ ] Bug fix
- [ ] Breaking change
- [ ] Performance improvement
- [ ] Refactoring
- [ ] Dependency update
- [ ] Documentation update
- [ ] Tests

## Which Components or Systems Does This Change Impact?
- [ ] Validator Node
- [ ] Full Node (API, Indexer, etc.)
- [ ] Move/Aptos Virtual Machine
- [ ] Aptos Framework
- [ ] Aptos CLI/SDK
- [ ] Developer Infrastructure
- [ ] Other (specify)

## Checklist
- [ ] I have read and followed the [CONTRIBUTING](https://github.com/aptos-labs/aptos-core/blob/main/CONTRIBUTING.md) doc
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I identified and added all stakeholders and component owners affected by this change as reviewers
- [ ] I tested both happy and unhappy path of the functionality
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: ecosystem/indexer-grpc/indexer-grpc-file-store/src/processor.rs [hunk 1]
```diff
@@ -238,7 +242,7 @@ impl Processor {
                 };
 
             // update next batch start version
-            batch_start_version = last_version + 1;
+            batch_start_version = e e + 1;
```
Comment (yuunlimm) [inline ecosystem/indexer-grpc/indexer-grpc-file-store/src/processor.rs:245]: what does this do?

Comment (larry-aptos) [reply ecosystem/indexer-grpc/indexer-grpc-file-store/src/processor.rs:245]: wait what..

Comment (larry-aptos) [reply ecosystem/indexer-grpc/indexer-grpc-file-store/src/processor.rs]: fixed; good catch!

**Review by yuunlimm — APPROVED** [review]
🚀

**Review by yuunlimm — APPROVED** [review]

File: ecosystem/indexer-grpc/indexer-grpc-file-store/src/processor.rs [hunk 2]
```diff
@@ -132,8 +133,17 @@ impl Processor {
             while start_version + (FILE_ENTRY_TRANSACTION_COUNT) < cache_worker_latest {
                 batches.push(start_version);
                 start_version += FILE_ENTRY_TRANSACTION_COUNT;
+                if batches.len() >= MAX_CONCURRENT_BATCHES {
+                    break;
+                }
             }
 
+            tracing::info!(
```
Comment (yuunlimm) [inline ecosystem/indexer-grpc/indexer-grpc-file-store/src/processor.rs:141]: qq: wondering if we need to import info

**Review by jillxuu — APPROVED** [review]

**Pull request merged**